staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/generic/webhook.go

```diff
@@ -207,6 +218,24 @@ func (a *Webhook) ShouldCallHook(h webhook.WebhookAccessor, attr admission.Attri
 		return nil, matchObjErr
 	}
 
+	matchConditions := h.GetMatchConditions()
+	if len(matchConditions) > 0 {
+		versionedAttr, err := v.VersionedAttribute(attr, o, invocation.Kind)
+		if err != nil {
+			return nil, apierrors.NewInternalError(err)
+		}
+
+		matcher := h.GetCompiledMatcher(a.filterCompiler, a.authorizer)
+		matchesCondition, matchConditionName, err := matcher.Match(ctx, versionedAttr, nil)
+		if err != nil {
+			// Ignore error and assume that an error is matching rather than skipping the webhook invocation on error
+			klog.V(5).Infof("Error evaluating match conditions for webhook %v, executing webhook call", h.GetName(), matchConditionName)
```

Comment:
It looks like there are more formatting variables than formatting directives here.